helm/alfresco-content-services/templates/deployment-repository.yaml

```diff
@@ -103,6 +103,12 @@ spec:
 {{ toYaml .Values.repository.extraVolumeMounts | indent 10 }}
           {{- end }}
           {{- end }}
+          startupProbe:
+            httpGet:
+              path: /alfresco/api/-default-/public/alfresco/versions/1/probes/-ready-
+              port: {{ .Values.repository.image.internalPort }}
+            periodSeconds: {{ .Values.repository.startupProbe.periodSeconds }}
+            failureThreshold: {{ .Values.repository.startupProbe.failureThreshold }}
           readinessProbe:
```

Comment:
see my comment in conversation.
tl;dr: I think this should not go to the main container